requirements.txt

```diff
@@ -0,0 +1,2 @@
+numpy
+scipy
```

Comment:
This should definitely fix a version. If unsure, use the major version that you are using